os/fs/smartfs/smartfs_utils.c

```diff
@@ -2284,15 +2284,11 @@ int smartfs_invalidate_old_entry(struct smartfs_mountpt_s *fs, uint16_t logsecto
 		if (firstparent->time > secondparent->time) {
 			ret = smartfs_invalidateentry(fs, secondparent->parentsector, secondparent->parentoffset);
 			if (ret == OK) {
-				sq_rem((FAR sq_entry_t *)secondparent, entry_queue);
```

Comment:
Not adding the last extra node to the queue now?

Reply:
It came from smartfs_scan_entry, nothing need to be added here.